include/generic/SugarWidgets/SugarWidgetFieldname.php

```diff
@@ -55,6 +55,7 @@ public function __construct(&$layout_manager)
     {
         parent::__construct($layout_manager);
         $this->reporter = $this->layout_manager->getAttribute('reporter');
+        $this->sugarWidgetFieldId = new SugarWidgetFieldId($layout_manager);
```

Comment:
Hi @TwizzX17, 

Thank you for your PR.

@TwizzX17 @serhiisamko091184 

The `sugarWidgetFieldId` is not declared on the class. It should be declared, otherwise we can get warnings/notices.

Reply:
Hi @clemente-raposo,
Of course - I've declared the property as a protected property. Do you agree with the implementation?